scripts/biomedical/Human-GEM_Human1/README.md

```diff
@@ -0,0 +1,170 @@
+# Importing Human-GEM Human1 data
+
+## Table of Contents
+
+1. [About the Dataset](#about-the-dataset)
+    1. [Download URL](#download-url)
+    2. [Database Overview](#database-overview)
+    3. [Schema Overview](#schema-overview)
+    4. [Notes and Caveats](#notes-and-caveats)
+    5. [License](#license)
+2. [About the Import](#about-the-import)
+    1. [Scripts](#scripts)
+    2. [tMCF files](#tMCF-files)
+    3. [Example](#example)
+    
+    
+## About The Dataset
+
+### Download URL
+
+The HumaGEM-Human1 data files (gense.tsv, metabolites.tsv, reactions.tsv, groups.tsv, geneRoles.tsv, reactantRoles.tsv, productRoles.tsv, groupMemberships.tsv), which were generated by extracting data from [Human-Gem.xml](https://github.com/SysBioChalmers/Human-GEM/blob/master/model/Human-GEM.xml) file in [Human-GEM repository](https://github.com/SysBioChalmers/Human-GEM), were obtained from [this repository](https://github.com/paul-shannon/bio-gdb/tree/main/forSam/v1). <br>
+An interactive web portal for viewing Human1 data can be found [here](https://metabolicatlas.org/)
+
+The recon-store-metabolites-1.tsv file can be downloaded from [Virtual Metabolic Human database](https://www.vmh.life/#human/all): Choose "4138 metabolites found", set page site to 4138 and download the file. The hmdb.csv file was sparsed from xml file downloaded from the "All Metabolites" dataset which can be found [here]((https://hmdb.ca/downloads)
+
+### Database Overview 
+
+This directory stores scripts and data for generating csv files which can be combined with the provided tMCF to ingest Human1 data to Data Commons knowledge graph. 
+
+#### Data files <br>
+
+Human1 data files can be found in ./data folder 
+
+* Entities:
+
+|File| Description|
+---------|-----------
+|genes.tsv| containing information about genes|
+|metabolites.tsv| containing informaion about metabolites|
+|reactions.tsv| containing information about reactions|
+|groups.tsv| containing information about metabolites classification/metabolic subsytems|
+
+* Relationships:
+
+|File| Description|
+---------|-----------
```

Comment:
Missing test data and test scripts. These should be added, but don't block the import.